modules/comment/comment.service.js

```diff
@@ -1,28 +1,28 @@
-const Comments = require('./comments.model');
+const Comments = require('./comment.model');
 
 class CommentsService {
   getAllComments() {
     return Comments.find();
   }
 
-  getCommentsById(id) {
+  getCommentById(id) {
     return Comments.findById(id);
   }
 
   getAllCommentsByProduct(id) {
     return Comments.find({ product: id });
   }
 
-  updateCommentsById(id, comments) {
+  updateComment(id, comments) {
     return Comments.findByIdAndUpdate(id, comments);
   }
 
-  addComments(data) {
+  addComment(data) {
     const comments = new Comments(data);
```

Comment:
Please, add check that product that is provided by client exists. If it doesn't exist throw 400 error

Reply:
Error handling will be in the next pull request